src/update/UpdateManager.cpp

```diff
@@ -65,47 +87,42 @@ void UpdateManager::getUpdates()
 
     response->setRequestFinishedCallback([response, this](audacity::network_manager::IResponse*) {
 
-        wxFrame* parent = FindProjectFrame(GetActiveProject());
-        wxASSERT(parent);
-
-        if (!parent) return;
-
         if (response->getError() != audacity::network_manager::NetworkError::NoError)
         {
-            ShowExceptionDialog(parent,
-                XO("Error checking for update"),
-                XO("Unable to connect to Audacity update server."),
+            wxTheApp->CallAfter([] {ShowExceptionDialog(nullptr,
```

Comment:
In this and the other lambdas, you use nullptr for parents of dialogs.

You might safely use `FindProjectFrame(GetActiveProject())` instead.  It would be sure to center the dialog on the main project window that the user is looking at.

This may be an unimportant, small advantage.

Reply:
When I tested with `wxTheApp` using it was centered too. Just I want avoid from project is not created (on macOS when menu shown only) by `wxTheApp` using.